common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -1481,6 +1481,9 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     HIVEQUERYNAME ("hive.query.name", null,
         "This named is used by Tez to set the dag name. This name in turn will appear on \n" +
         "the Tez UI representing the work that was done."),
+    HIVETEZJOBNAME("tez.job.name", null,
```

Comment:
The second value here, I believe is a 'default' value.  The default should be `HIVE-%s`